.gitignore

```diff
@@ -1 +1,3 @@
 .env
+Formulas/symflow@.rb
+Formulas/sym@.rb
```

Comment:
Context: `symflow@.rb` is invalid, ignoring to ensure that they never get pushed.